content/blog/2022/2022-02-14-spring-shell.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Create Command-line Applications with Spring Shell"
+categories: ["Spring"]
+date: 2022-02-26 00:00:00 +1100
+authors: ["cercenazi"]
+description: "Introduction to Spring Shell and how to create a simple command line application."
+image: images/stock/0119-cat-1920-1282.jpg
```

Comment:
Haha! If you can tell me the connection between the cat and Spring Shell, I'll leave this picture in here :). Otherwise, I'll choose another if you don't mind.

Reply:
I chose it because it's cute and might attract readers (click bait :D) 
I'll look for something more SpringShell'y though :D

Reply:
how about this? https://unsplash.com/photos/dln9ARqL6iQ